example/brunel/brunel.cpp

```diff
@@ -351,10 +351,9 @@ void add_subset(cell_gid_type gid,
     // Exclude ourself
     std::set<cell_gid_type> seen{gid};
     std::mt19937 gen(gid + 42);
-    std::uniform_int_distribution<cell_gid_type> dis(start, end - 1);
-
     while(m) {
-        cell_gid_type val = dis(gen);
+        // simple, compiler independent int in range
```

Comment:
Since this is just an example model, I guess it's fine. But there might be a bias introduced by this approach (see [here](https://ericlippert.com/2013/12/16/how-much-bias-is-introduced-by-the-remainder-technique/)). So adding a warning note as a comment would be good.